gvm/protocols/gmpv9/gmpv9.py

```diff
@@ -75,23 +75,6 @@ def _check_event(
                     condition.name, event.name
                 )
             )
-        if method not in (
-            AlertMethod.SCP,
-            AlertMethod.SEND,
-            AlertMethod.SMB,
-            AlertMethod.SNMP,
-            AlertMethod.SYSLOG,
-            AlertMethod.EMAIL,
-            AlertMethod.START_TASK,
-            AlertMethod.HTTP_GET,
-            AlertMethod.SOURCEFIRE_CONNECTOR,
-            AlertMethod.VERINICE_CONNECTOR,
-            AlertMethod.TIPPINGPOINT,
-            AlertMethod.ALEMBA_VFIRE,
-        ):
-            raise InvalidArgument(
-                "Invalid method {} for event {}".format(method.name, event.name)
-            )
```

Comment:
I have removed this, as these are ALL existing Methods.
There are Checks before this function in create/modify Alert that check if the passed argument has the correct type, so this should always be true and this Exception will never be raised ....